installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -0,0 +1,134 @@
+EOF
+----
+
+. Remove the admin credential secret created using your local cloud credentials.
+This removal prevents your admin credential from ever being stored in the
+cluster:
++
+----
+$ rm mycluster/openshift/99_cloud-creds-secret.yaml
+----
+
+. Obtain the {product-title} release image your `openshift-install` binary is built
+to use:
++
+----
+$ bin/openshift-install version
+release image quay.io/openshift-release-dev/ocp-release:4.z.z-x86_64
+----
+
+. Locate all `CredentialsRequests` in this release image that target the cloud you
+are deploying on:
++
+----
+$ oc adm release extract quay.io/openshift-release-dev/ocp-release:4.z.z-x86_64 --to ./release-image
+----
+
+. Locate the `CredentialsRequests` in the extracted file:
++
+----
+$ grep -l "apiVersion: cloudcredential.openshift.io" * | xargs cat
+----
++
+[NOTE]
+====
+In a future {product-title} release, there will be a new `oc adm release`
+command to scan for the `CredentialsRequests` and display them.
+====
++
+This displays the details for each request. Remember to ignore any
+`CredentialsRequests` where the `spec.providerSpec.kind` does not match the cloud
+provider you are installing to.
++
+.Sample CredentialsRequest
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cloud-credential-operator-iam-ro
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: cloud-credential-operator-iam-ro-creds
+    namespace: openshift-cloud-credential-operator
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - effect: Allow
+      action:
+      - iam:GetUser
+      - iam:GetUserPolicy
+      - iam:ListAccessKeys
+      resource: "*"
+----
+
+. Create secret YAML files in the `openshift-install` manifests directory that you
+generated previously. The secrets must be stored in the namespace and name
+defined in each `request.spec.secretRef`. The format for the secret data varies
+for each cloud provider.
+
+. Proceed with cluster creation:
++
+----
+$ openshift-install create cluster --dir=mycluster
+----
++
+[IMPORTANT]
+====
+Before performing an upgrade, you might need to adjust your credentials if
+permissions have changed in the next release. In the future, the Cloud
+Credential Operator might prevent you from upgrading until you have indicated
+that you have addressed updated permissions.
+====
+
+include::modules/admin-credentials-root-secret-formats.adoc[leveloffset=+1]
+
+[id="manually-maintained-credentials-upgrade_{context}"]
+== Upgrades
+
+In a future release, improvements to the Cloud Credential Operator will prevent
+situations where a user might enter an upgrade that will fail because their
+manually maintained credentials have not been updated to match the
+`CredentialsRequests` in the upcoming release image. In the meantime, this is
+being  monitored to ensure no upgrade breaking credentials changes go in.
```

Comment:
I would drop the last sentence here, that's probably from our repo but not really relevant to users. Not even true actually, we're still trying to get the CI job ready for it.